examples/options/web/src/ui/theme.css

```
@@ -0,0 +1,628 @@
+  }
+}
+@media (max-width: 460px) {
+  .form-grid {
+    grid-template-columns: 1fr;
+  }
+}
+
+/* long, unbreakable hex (addresses, keys) must not force horizontal scroll */
+.mono-wrap {
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)